spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -74,5 +75,25 @@
         )
       end
     end
+
+    context 'with max targets reached' do
```

Comment:
I probably create different `context` to test edge values.. like
- when I have 9 Targets and try to create the number 10 (it should allow me)
- when I have 10 Target and try to create the number 11 (it should raise an error) --> this is the one you implemented I think